README.rst

```diff
@@ -40,6 +40,41 @@ Single-node influxdb, SSL for http frontend:
               key_file: /etc/influxdb/ssl/key.pem
               cert_file: /etc/influxdb/ssl/cert.pem
 
+Single-node influxdb with an admin user (only one admin user is supported
+currently):
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        enabled: true
+        http:
+          enabled: true
+          bind:
+            address: 0.0.0.0
+            port: 8086
+        admin:
+          enabled: true
+          bind:
+            address: 0.0.0.0
+            port: 8083
+          user:
+            name: root
+            password: secret
+
+Single-node influxdb with a new database and its non-admin user:
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        databases:
```

Comment:
I'm ok with it.
Or you can move grants under database (as they are per-database) instead of having new grants structure. But this one is also fine.